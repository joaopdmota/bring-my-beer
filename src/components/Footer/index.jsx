import React, { forwardRef } from 'react'
import Grid from '../Grid'
import Button from '../Button'
import styled, { css } from 'styled-components'

const Footer = styled.footer`
  padding: 8px 0;
  background-color: ${props => props.theme.mainColor};
  width: 100%;
  position: relative;
  a {
    color: ${props => props.theme.txt};
  }
  ${props =>
    props.shouldStick &&
    css`
      position: fixed;
      left: 0;
      right: 0;
      bottom: 0;
    `};
`

export default forwardRef((props, ref) => {
  return (
    <Footer ref={ref} {...props}>
      <Grid justify="space-between" container>
        <p>
          by{' '}
          <a href="http://github.com/venturalp">Guilherme Ventura de souza</a>
        </p>
      </Grid>
    </Footer>
  )
})
