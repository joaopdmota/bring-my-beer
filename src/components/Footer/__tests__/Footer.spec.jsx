import React from 'react'
import Footer from '../index'

describe('Footer component', () => {
  it('render footer', () => {
    const footer = <Footer />
    expect(footer).toMatchSnapshot()
  })
})
