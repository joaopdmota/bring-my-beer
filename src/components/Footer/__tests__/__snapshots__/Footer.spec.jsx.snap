// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component render footer 1`] = `<ForwardRef />`;
